spawn_operator/spawn_operator/lib/spawn_operator/controller/actor_host_controller.ex

```diff
@@ -1,21 +1,21 @@
 defmodule SpawnOperator.Controller.ActorHostController do
   require Bonny.API.CRD
 
-  use Bonny.ControllerV2,
-    for_resource:
-      Bonny.API.CRD.build_for_controller!(
-        group: "spawn-eigr.io",
-        scope: :Cluster,
-        versions: [SpawnOperator.Versions.Api.V1.ActorHost]
-      )
+  use Bonny.ControllerV2
 
-  use SpawnOperator.Handler.ActorHostHandler
+  # step Bonny.Pluggable.SkipObservedGenerations
```

Comment:
Feel free to uncomment these steps. Note however, that you have to uncomment the according lines in the version modules as well.